ouroboros-consensus-test/src/Test/Util/Serialisation/Roundtrip.hs

```diff
@@ -212,6 +216,27 @@ data WithVersion v a = WithVersion v a
 -- @('WithVersion' v a)@.
 type ArbitraryWithVersion v a = (Arbitrary (WithVersion v a), Eq a, Show a)
 
+instance ( Arbitrary (WithVersion (BlockNodeToClientVersion blk) (SomeSecond BlockQuery blk))
+         , SupportedNetworkProtocolVersion blk
```

Comment:
The code seems to compile without `SupportedNetworkProtocolVersion blk`. Do you think there's a reason to require it anyway?

Reply:
Ah yes. This is a vestigial change. I'll remove it.